### PR TITLE
Most used words links mentions and hashtags

### LINKED ISSUE #87: When finding words in Tweets, conflate hashtags with normal words
E.g. for `glasgow` we should see responses for `GlasGow` as well as `#glasgow` and `#GLASGOW` instead of it being a separate term.

### PR DESCRIPTION
**Pull request by Winwardo**
Fixes #87 and #86.

### CODE REVIEW

File: src/client/tweetAnalysis.spec.js [hunk 1]
```diff
@@ -30,6 +30,37 @@ describe('#TweetAnalysis', () => {
       mostFrequentWords(tweets).should.deep.equal(exampleFrequentWords);
     });
```
Comment (VasilyShelkov) [inline src/client/tweetAnalysis.spec.js:32]: So we're including URLs in the word count ?

Comment (Winwardo) [inline src/client/tweetAnalysis.spec.js:32]: I'd say we should, now that they're all unique, each one will only appear once anyway. Do you think we should remove them as they're not "words"?

Comment (VasilyShelkov) [inline src/client/tweetAnalysis.spec.js:32]: Nah let's keep them for now and I'll send an email to Fabio asking if he wants us to consider them as individual words or not :)

Comment (Winwardo) [inline src/client/tweetAnalysis.spec.js:32]: Along with a million other questions? XD

**Pull request merged**